wbooks/src/app/screens/BookDetail/components/CommentTable/component/CommentItem/index.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import { View, Image, Text } from 'react-native';
+import PropTypes from 'prop-types';
+
+import styles from './styles';
+
+class CommentItem extends Component {
```

Comment:
replace this class with an arrow function component:
```
const CommentItem = () => (
...
);
```

Reply:
done